llumnix/llumlet/migration_coordinator.py

```diff
@@ -67,6 +67,7 @@ async def migrate_out_onestage(self, migrate_in_ray_actor: "ray.actor.ActorHandl
             # last stage migration, stop inference, transfer all blocks
             migration_status = MigrationStatus.FINISHED_DONE
             self.backend_engine.remove_running_request(migrate_out_request.request_id)
+            await self.backend_engine.send_request_group(migrate_in_ray_actor, migrate_out_request.request_id)
```

Comment:
similarly here, I don't think it's a good choice to have concepts like request group in our interface methods